styles/main.css

```diff
@@ -78,21 +91,29 @@ header nav {
 ol li {
   list-style-type: upper-roman;
 }
-ul li {
+header nav ul li {
   display: inline;
 }
 
 header nav ul {
   margin: 0;
-  padding-top: 1em;
-  background-image: url(link.png);
-  background-repeat: no-repeat;
-  background-size:10px 10px;
-  background-position:left center ;
```

Comment:
This applies to the header list.  Yo uwere supposed to have these declarations in a rule in the internal style for the links page (see above)